fe/src/main/java/org/apache/doris/planner/ScanNode.java

```diff
@@ -37,6 +50,9 @@
     protected final TupleDescriptor desc;
     protected Map<String, PartitionColumnFilter> columnFilters;
     protected String sortColumn = null;
+    protected Map<String, SlotDescriptor> slotDescByName = Maps.newHashMap();
```

Comment:
If you want to unify BrokerScanNode and StreamScanNode, I think you'd better to make another class which is subclass of ScanNode. Because not all ScanNode has these properties, they only are used in loading.